Composer/packages/server/src/controllers/publisher.ts

```diff
@@ -83,6 +83,7 @@ export const PublishController = {
         // set status and return value as json
         res.status(results.status).json(response);
       } catch (err) {
+        console.error('Publishing error: ' + err.message);
```

Comment:
this is not exactly what I meant, since this is just going to say 
"failed to run dotnet build"

we can attach to the spawned child process and output the errors it generates to get more detail.

Reply:
the actual `exec`  happens inside the localPublish plugin.